### PR TITLE
Add info about device to the error report used in Crash events

### PR DESCRIPTION
**Pull request by evgeny-pol**
Add info about device to the error report used in Crash events

### CODE REVIEW

File: Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesInternal.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@
 using System.Collections.Generic;
 using System.Runtime.InteropServices;
 using Microsoft.AppCenter.Unity.Crashes.Models;
+using UnityEngine;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesInternal.cs]: It looks like unused

File: Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.h [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+extern "C" const char* app_center_unity_device_sdk_name(void* device);
+extern "C" const char* app_center_unity_device_sdk_version(void* device);
+extern "C" const char* app_center_unity_device_model(void* device);
+extern "C" const char* app_center_unity_device_oem_name(void* device);
+extern "C" const char* app_center_unity_device_os_name(void* device);
+extern "C" const char* app_center_unity_device_os_version(void* device);
+extern "C" const char* app_center_unity_device_os_build(void* device);
+extern "C" const int app_center_unity_device_os_api_level(void* device);
+extern "C" const char* app_center_unity_device_locale(void* device);
+extern "C" const int app_center_unity_device_time_zone_offset(void* device);
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.h]:
```suggestion
extern "C" int app_center_unity_device_time_zone_offset(void* device);
```

File: Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.mm [hunk 3]
```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+#import "NSStringHelper.h"
+#import "DeviceHelper.h"
+#import <AppCenter/AppCenter.h>
```
Comment (MatkovIvan) [inline Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.mm:5]: Nitpick: Reorder.
1. `#import <*>`
2. `#import "*"`
+ Alphabetically

File: Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesInternal.cs (hunk 1 above)
Comment (evgeny-pol) [reply Assets/AppCenter/Plugins/AppCenterSDK/Crashes/iOS/CrashesInternal.cs]: Done, unused `Using` statements has been removed

File: Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.h (hunk 2 above)
Comment (evgeny-pol) [reply Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.h]: Done, `const` attribute removed

File: Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.mm (hunk 3 above)
Comment (evgeny-pol) [reply Assets/AppCenter/Plugins/iOS/Core/Utility/DeviceHelper.mm:5]: Done

**Review by max-mironov — APPROVED** [review]

**Pull request merged**